user-guide/Advanced_Functionality/DataMiner_Systems/Setting_up_a_new_DMS/Setting_up_a_new_DMS_in_DataMiner_Cube.md

```diff
@@ -19,20 +19,15 @@ uid: Setting_up_a_new_DMS_in_DataMiner_Cube
 
 1. For each DMA you want to add to the DMS, do the following:
 
-   1. Open DataMiner Cube on the DMA you want to add.
-
-   1. Go to *Apps* > *System Center* > *Agents.*
-
-   1. In the *Manage* tab, select the DataMiner Agent to which you are connected (it is marked by an icon with a small “1”).
-
-   1. In the pane on the right, next to *Cluster*, enter the same DMS name as you specified for the master DMA, and then click *Add Cluster* (prior to DataMiner 10.0.13) or *Join cluster* (from DataMiner 10.0.13 onwards).
-
    1. Open DataMiner Cube on the primary DMA, and navigate to the *Agents* > *Manage* tab as before.
 
    1. In the lower left corner, click *Add*.
 
    1. Enter the IP of the DMA you want to add and click *Add*.
 
+> [!NOTE]
```

Comment:
Similar to the comment above, I'd recommend combining this with the note below. We usually try to avoid adding several note blocks close to one another and combine the notes into one block instead.